docs/release_notes/v1.8.0.adoc

```diff
@@ -0,0 +1,43 @@
+[id="{p}-release-notes-v1.8.0"]
+=== Elastic Cloud Control (ecctl) v1.8.0
+++++
+<titleabbrev>v1.8.0</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.8.0-changelog,Changelog>>
+
+Welcome to the v1.8.0 release of {n}. This version brings new features, some breaking changes, and bug fixes.
+
+Elastic Cloud Control (ecctl) is Elastic’s CLI to manage Elasticsearch Service (ESS) and Elastic Cloud Enterprise (ECE).
+
+Download the release binaries:
+
+[%hardbreaks]
+https://download.elastic.co/downloads/ecctl/1.8.0/ecctl_1.8.0_darwin_amd64.tar.gz[ecctl_1.8.0_darwin_amd64.tar.gz]
+https://download.elastic.co/downloads/ecctl/1.8.0/ecctl_1.8.0_linux_32-bit.deb[ecctl_1.8.0_linux_32-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.8.0/ecctl_1.8.0_linux_32-bit.rpm[ecctl_1.8.0_linux_32-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.8.0/ecctl_1.8.0_linux_386.tar.gz[ecctl_1.8.0_linux_386.tar.gz]
+https://download.elastic.co/downloads/ecctl/1.8.0/ecctl_1.8.0_linux_64-bit.deb[ecctl_1.8.0_linux_64-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.8.0/ecctl_1.8.0_linux_64-bit.rpm[ecctl_1.8.0_linux_64-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.8.0/ecctl_1.8.0_linux_amd64.tar.gz[ecctl_1.8.0_linux_amd64.tar.gz]
+
+[float]
+[id="{p}-release-notes-v1.8.0-changelog"]
+==== Changelog
+// The following section is autogenerated via git
+
+[%hardbreaks]
+https://github.com/elastic/ecctl/commit/512d20a[512d20a] fix(deps): update go-openapi packages (https://github.com/elastic/ecctl/pull/554[#554])
```

Comment:
The only comment here is the use of a capital letter after the colon. Might not make sense to suggest the fix, as this is a generated file, right?